internify/src/components/organisms/styles/AuthNavbar.css

```diff
@@ -0,0 +1,65 @@
+.auth_navbar_container {
+
+    height: 80px;
+    padding-top: 20px;
+}
+
+
+.auth_navbar_link {
+    text-decoration: none;
+    color: black;
+  }
+
+  .auth_navbar_item {
+    float: right;
+  }
+
+  .auth_navbar_links {
+
+    position: absolute;
```

Comment:
this is concerning, we don't want to be using `position: absolute` whenever possible. If this is for the links on the navbar you should be doing something like:
```
display: flex;
flex-direction: row-reverse;
```
instead. The only thing that should be absolute is anything that needs to be strictly in the center of the screen. This is just so that we don't run into styling issues in the future, especially if we want to add more elements to this particular container

Reply:
noted! will fix